e2eAIOK/DeNas/scores/transformer_proxy.py

```diff
@@ -192,18 +211,19 @@ def do_compute_nas_score_transformer(model_type, model, resolution, batch_size,
         input = torch.randn(size=[batch_size, 3, resolution, resolution],  dtype=dtype)
         disversity_score_list = compute_diversity_score(model_type, model, input)
     elif model_type == "bert":
-        
-        max_seq_length = resolution
-        input_ids = [[9333-id] * max_seq_length for id in range(batch_size)]
-        input_masks = max_seq_length * [1]
-        input_segments = max_seq_length * [0]
+        input_ids = [[9333-id] * resolution for id in range(batch_size)]
+        input_masks = resolution * [1]
+        input_segments = resolution * [0]
         input_ids = torch.tensor(input_ids, dtype=torch.long)
         input_masks = torch.tensor([input_masks]*batch_size, dtype=torch.long)
         input_segments = torch.tensor([input_segments]*batch_size, dtype=torch.long)
```

Comment:
looks like bert also can use `input_constructor` to generate model input, can we replace it?